network-boot-lab.md

```diff
@@ -4,17 +4,17 @@
 
 Complete the instructions described in this lab.
 
-The student must send:
-1. A `zip file` containing the final folder after completing this tutorial and just removing the `data` folder.
+The student must send (--> where?):
+1. A `zip file` containing the current directory? after completing this tutorial and just removing the `data` folder.
```

Comment:
It isn't necessarily the current directory, it could be another, it just needs to refer to the folder where the student did the lab

Reply:
I'm gonna replace the suggestion with `the resulting folder`